docs/install.rst

```diff
@@ -322,22 +323,20 @@ The app has been tested with IE11, Mozilla FireFox 56 and Google Chrome v72.
 +-----------------------------------+----------------+
 | APIC logical devices              | 60             | 
 +-----------------------------------+----------------+ 
-| Per BIG-IP nodes members          | 4              | 
+| Per BIG-IP nodes members          | 100            | 
```

Comment:
is this 'nodes and members' or 'node's members' (possessive)?  If not either one of those, probably should be 'node members'